node/config.go

```diff
@@ -190,7 +187,7 @@ type FlagsYAML struct {
 	BootstrapIPs                            *string  `yaml:"bootstrap-ips,omitempty"`
 	BootstrapIDs                            *string  `yaml:"bootstrap-ids,omitempty"`
 	BootstrapBeaconConnectionTimeout        *string  `yaml:"bootstrap-beacon-connection-timeout,omitempty"`
-	DBEnabled                               *bool    `yaml:"db-enabled,omitempty"`
+	DBType                                  *bool    `yaml:"db-type,omitempty"`
```

Comment:
Shouldn't this be a string instead of a boolean?

Reply:
wow can't believe I missed that. I guess it was working when I was running it locally because I don't use the yaml based config.